content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -753,15 +754,59 @@ rules:
 
 See [Authorization Overview](/docs/reference/access-authn-authz/authorization/).
 
-## Server Side Apply
+## Updating existing resources {#patch-and-apply}
+
+You can overwrite (**update**) an existing object using an HTTP PUT. It is
+the client's responsibility when sending a PUT to specify the `resourceVersion`,
+so that the API server can detect lost updates. In the event that the resource has
+changed (the `resourceVersion` the client provided is stale), the API server
+returns a 409 conflict response.
+
+As an alternative, the client can send an instruction to the API server to
+**patch** an existing resource, or to **apply** a set of changes.
+
+The Kubernetes API supports four different PATCH operations, determined by their
+corresponding HTTP `Content-Type` header:
+
+`application/apply-patch+yaml`
+: Server Side Apply YAML (a Kubernetes-specific extension, based on YAML).
+  All JSON documents are valid YAML, so you can also submit JSON using this
+  media type. See [Server Side Apply serialization](/docs/reference/using-api/server-side-apply/#serialization)
+  for more details.  
+  To Kubernetes, this is an **apply** operation.
+
+`application/json-patch+json`
+: JSON Patch, as defined in [RFC6902](https://tools.ietf.org/html/rfc6902).
+  A JSON patch is a sequence of operations that are executed on the resource;
+  for example `{"op": "add", "path": "/a/b/c", "value": [ "foo", "bar" ]}`.  
+  To Kubernetes, this is a **patch** operation.
+
+`application/merge-patch+json`
+: JSON Merge Patch, as defined in [RFC7386](https://tools.ietf.org/html/rfc7386).
+  A JSON Merge Patch is essentially a partial representation of the resource.
+  The submitted JSON is combined with the current resource to create a new one,
+  then the new one is saved.  
+  To Kubernetes, this is a **patch** operation.
+
+`application/strategic-merge-patch+json`
```

Comment:
Note that this is not available to CRs, only built-in types.

Reply:
Good to know. BTW is that due to a bug, a planned feature, something else?

Reply:
I don't know the exact history behind that, but we basically never found the time/envy to make it work for CRs, and then server-side apply started so it felt like a waste of time.

Reply:
Also, schemas wasn't readily available initially to make these things work, so the work was never started.

Reply:
strategic-merge-patch relies on complex perfectly-matching logic existing both in the client and the server (in the client to compute the patch, and in the server to apply the patch). in practice, this made it exceptionally hard (or impossible) to fix bugs in strategic merge patch application or use without breaking old clients against new servers, or new clients against old servers (xref https://github.com/kubernetes/kubernetes/pull/107549#discussion_r785029380)

We didn't really want to import those problems to custom resources.